src/pybind/mgr/prometheus/module.py

```diff
@@ -1740,6 +1756,40 @@ def get_all_daemon_health_metrics(self) -> None:
                 self.metrics[path].set(health_metric['value'], labelvalues=(
                     health_metric['type'], daemon_name,))
 
+    def get_cephadm_container_metrics(self) -> None:
+        """
+        Populate cephadm_container_cpu_percentage and cephadm_container_memory_bytes
+        from the orchestrator.
+        """
+        if not self.orch_is_available():
+            return
+
+        try:
+            daemons = raise_if_exception(self.list_daemons())
+            for daemon in daemons:
```

Comment:
a question here, won't this be a scale issue? fetching and iterating over all the daemons that are there....? shouldn't these kind of things handled by the ceph-exporter?